kuka_rox_hardware_interface/src/rox_hardware_interface.cpp

```diff
@@ -23,7 +23,7 @@
 #include <string>
 #include <vector>
 
-#include "nanopb-helpers/nanopb_serialization_helper.h"
```

Comment:
did it not build with the original?

Reply:
Nope, this is actually an issue from last week. I can change this line to make the CI happy temporarily